server/src/main/java/org/elasticsearch/action/admin/indices/forcemerge/ForceMergeRequest.java

```diff
@@ -53,20 +55,36 @@
     private boolean onlyExpungeDeletes = Defaults.ONLY_EXPUNGE_DELETES;
     private boolean flush = Defaults.FLUSH;
 
+    private static final Version FORCE_MERGE_UUID_VERSION = Version.V_8_0_0;
+
+    public static final String FORCE_MERGE_UUID_NA_VALUE = "_na_";
```

Comment:
I figured it's a nice thing to just used `_na_` instead of `null` to keep the serialization BwC simple.